arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+            <message fade-in="0.0s" stay="0.7s" fade-out="0.3s" subtitle="`6`lRound `e{amount}">
+                <title>`2Go</title>
+                <replacements>
+                    <decimal id="amount" value="round_number"/>
+                </replacements>
+            </message>
+        </action>
+    </trigger>
+</actions>
+<structures>
+    <structure id="course" air="true" clear="false">
+        <region>
+            <cuboid min="-11,0,-43" max="11,21,43"/>
+        </region>
+    </structure>
+    <dynamic structure="course">
+        <trigger>
+            <filter id="reset1-state"/>
+        </trigger>
+    </dynamic>
+</structures>
+<regions>
+    <!-- course boundaries -->
+    <below id="void" y="0"/>
+    <above y="23" id="ceiling"/>
+    <union id="no-go">
+        <cuboid id="zone-1a" min="12,23,20" max="13,1,44"/>
+        <cuboid id="zone-1b" min="12,23,-43" max="13,1,-19"/>
+        <cuboid id="zone-2a" min="-11,1,-19" max="-12,23,-43"/>
+        <cuboid id="zone-2b" min="-11,1,44" max="-12,23,20"/>
+    </union>
+    <negative id="not-buildable">
+        <cuboid id="buildable" min="-10,0,-42" max="11,21,43"/>
+    </negative>
+    <cylinder id="original-block-protection" base=".5,0,.5" radius="100" height="176"/>
+    <!-- spawn locations -->
+    <point id="team-one-spawn-point">${team-one-spawn}</point>
+    <point id="team-two-spawn-point">${team-two-spawn}</point>
+    <union id="spawn-protections">
+        <cuboid id="team-one-spawn-protection" min="-3,20,-42.5" max="4,66,-36.5"/>
+        <cuboid id="team-two-spawn-protection" min="-3,20,43.5" max="4,66,36.5"/>
+    </union>
+    <union id="cages">
+        <cylinder id="team-one-spawn-cage" base="${team-one-spawn}" height="4" radius="4"/>
+        <cylinder id="team-two-spawn-cage" base="${team-two-spawn}" height="4" radius="4"/>
+    </union>
+    <!-- bed locations -->
+    <cuboid id="bed-red-1" min="0,10,-40" max="1,11,-41"/>
+    <cuboid id="bed-red-2" min="0,10,-41" max="1,11,-42"/>
+    <cuboid id="bed-blue-1" min="0,10,41" max="1,11,42"/>
+    <cuboid id="bed-blue-2" min="0,10,42" max="1,11,43"/>
+    <!-- applies -->
+    <apply enter="never" region="ceiling" message="You may not climb higher!"/>
+    <apply enter="never" region="no-go" message="You may not leave the course!"/>
+    <apply block="original-block-protection-filter" message="You can't modify that block!"/>
+    <apply block-place="never" region="not-buildable" message="You may not build in this area!"/>
+    <apply block-break="red-bed-protect-1" region="bed-red-1" message="You may not break your own bed!"/>
+    <apply block-break="red-bed-protect-2" region="bed-red-2" message="You may not break your own bed!"/>
+    <apply block-break="blue-bed-protect-1" region="bed-blue-1" message="You may not break your own bed!"/>
+    <apply block-break="blue-bed-protect-2" region="bed-blue-2" message="You may not break your own bed!"/>
```

Comment:
As suggested above, i'd update these to be:
```suggestion
    <apply block-break="deny-red" region="bed-red" message="You may not break your own bed!"/>
    <apply block-break="deny-blue" region="bed-blue" message="You may not break your own bed!"/>
```

Reply:
Done.